wsapi.go

```diff
@@ -202,8 +222,8 @@ func (s *Session) event(messageType int, message []byte) (err error) {
 			if s.OnReady != nil {
 				s.OnReady(s, st)
 			}
-			go s.Heartbeat(st.HeartbeatInterval)
 		}
+		go s.heartbeat(st.HeartbeatInterval)
```

Comment:
I can't see any reason to keep Heartbeat as a public method, we removed the need to call it manually previously, and it is cleaned up with the Listen() loop on Close().